versioned_docs/version-8.7/self-managed/identity/deployment/configuration-variables.md

```diff
@@ -132,17 +132,92 @@ steps:
+| `IDENTITY_TENANTS_0_TENANTID`                | The tenant ID. The `<default>` tenant is automatically created during Identity startup.                                                                                                                                                            | `<default>`   |
+| `IDENTITY_TENANTS_0_MEMBERS_0_TYPE`          | The type of member to add to the tenant, one of `GROUP`, `USER`, or `APPLICATION`. Additional members can be added by incrementing the second `0` value (for example, `IDENTITY_TENANTS_0_MEMBERS_1_TYPE`.)                                        | `APPLICATION` |
+| `IDENTITY_TENANTS_0_MEMBERS_0_APPLICATIONID` | If adding an **application** as a member of the tenant, the application ID. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_APPLICATIONID`). | `zeebe`       |
+| `IDENTITY_TENANTS_0_MEMBERS_0_USERNAME`      | If adding a **user** as a member of the tenant, the username. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_USERNAME`).                    | `user-name`   |
+| `IDENTITY_TENANTS_0_MEMBERS_0_GROUPNAME`     | If adding a **group** as a member of the tenant, the name of the group. If additional members have been added to the tenant, ensure the second `0` value matches that of its type (for example, `IDENTITY_TENANTS_0_MEMBERS_1_GROUPNAME`).         | `group-name`  |
+| `IDENTITY_ENVIRONMENT_TENANTS_0_NAME`        | Needs description.                                                                                                                                                                                                                                 | Default       |
+| `IDENTITY_ENVIRONMENT_TENANTS_0_TENANTID`    | Needs description.                                                                                                                                                                                                                                 | `<default>`   |
+
+When converting the above variables for use in the Helm chart, ensure multiple tenants or members are added as a list. For example, a valid `application.yaml` configuration might look like the following:
+
+```
+identity:
+   tenants:
+    - name: Tenant Name
+      tenantId: tenant-name
+      members:
+        - type: GROUP
+          group-name: "Test Group"
+        - type: USER
+          username: test-user
+        - type: APPLICATION
+          application-id: test-application
+```
+
+## Configure clients
+
+The following variables can be used to configure [clients](/self-managed/identity/user-guide/additional-features/incorporate-applications.md). To add additional clients, increment the `0` value for each variable (for example, `KEYCLOAK_CLIENTS_1_NAME`).
```

Comment:
Is this the appropriate link? Is a client any of these applications?

Reply:
To the best of my knowledge that's correct. The applications displayed in the Identity UI match the clients in Keycloak.